frontend/src/Metamaps/Control.js

```diff
@@ -20,6 +20,21 @@ const Control = {
     node.setData('dim', 30, 'current')
     Selected.Nodes.push(node)
   },
+  selectNeighbors: function() {
+    if(Selected.Nodes.length > 0){
+      //For each selected node, select all connected node and the synapses too
+      Selected.Nodes.forEach((item) => {
+        if(Visualize.mGraph.graph.getNode(item.id).adjacencies){
+          for(var adjID in Visualize.mGraph.graph.getNode(item.id).adjacencies){
```

Comment:
I prefer const or let to var, because var is scoped to the function which is confusing. Const is probably best, and then it'll stay scoped to the loop